lib/fauna/objects.rb

```diff
@@ -8,11 +8,35 @@ class Ref
     attr_accessor :value
 
     ##
-    # Creates a Ref object from a string.
-    #
-    # +ref+:: A ref in string form.
-    def initialize(ref)
-      self.value = ref
+    # :call-seq:
```

Comment:
* Restore the other documentation
* Put blank comment lines in between sections for readability.
* Do the `:call-seq:` elsewhere that needs it